src/entities/UserFavorite/schema.ts

```diff
@@ -0,0 +1,36 @@
+import schema from "decentraland-gatsby/dist/entities/Schema/schema"
+
+export const updateUserFavoriteParamsSchema = schema.params({
+  place_id: {
+    type: "string",
+    format: "uuid",
+    description: "Place ID",
+  },
+})
+
+export const updateUserFavoriteBodySchema = schema({
+  type: "object",
+  description: "user favorite query needed",
+  additionalProperties: false,
+  required: ["favorites"] as const,
+  properties: {
+    favorites: {
+      type: "boolean",
+      description: "True if the place is selected as favorite",
+    },
+  },
+})
+
+export const userFavoriteSchema = schema({
+  type: "object",
+  required: [],
+  properties: {
+    total: {
```

Comment:
also, update this, please

```suggestion
    favorites: {
```